galata/package.json

```
@@ -0,0 +1,81 @@
+{
+  "name": "@jupyterlab/galata",
+  "version": "4.3.0-alpha.5",
```

Should this be updated for consistency?

```suggestion
  "version": "3.3.0-alpha.6",
```

---

Based on the current version on master:

https://github.com/jupyterlab/jupyterlab/blob/9c592fcee69915a2581ed4bdfeaddbda8bf5acb3/dev_mode/package.json#L3

---

I bumped Galata to v4 as the current version is v3 but comes with a totally different approach. So we should update this to `4.3.0-alpha.6`. this gives us the room to release `4.0.0` with JLab v3.2.

---

Or `4.0.0-alpha.6`? I was wondering why the version on master is `3.3.0-alpha.6` while the release is `4.0.0a6` and thought it could have been a glitch in the automated release process?

---

> Or 4.0.0-alpha.6?

The idea is to publish 4.0.0 when we release JupyterLab 3.2.x to make it available for other projects quickly.

> I was wondering why the version on master is 3.3.0-alpha.6 while the release is 4.0.0a6 and thought it could have been a glitch in the automated release process?

I do not follow this question. Could you point me towards the place where that version appears?

---

> Could you point me towards the place where that version appears?

This snippet above:

https://github.com/jupyterlab/jupyterlab/blob/9c592fcee69915a2581ed4bdfeaddbda8bf5acb3/dev_mode/package.json#L3

Most of the time until now the Python and npm versions were pretty much in sync (`3.0.0a6` and `3.0.0-alpha.6` for example). Which is why `3.3.0-alpha.6` looked confusing at first for the corresponding `4.0.0a6` Python version. And thought it could have been related to an issue during the version bump of the releases.

---

It was decided to not follow Python major version any longer (using semver rules only) to avoid all extension authors to carry actions to be compatible to the next major JupyterLab version (except if the packages they depend on require it).

The bump to `3` as minor version number is a consequence of the previous new rule and to let some space for new minor bump for 3.2.x branch.

So no problem with the releaser here :tada:

---

Ah ok makes sense then.

Thanks for the clarifications :+1: